runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -151,7 +151,7 @@ void addLog(final String eventName, final int actorLevel, LootTrackerItemEntry[]
 		updateOverall();
 
 		// Create reset menu
```

Comment:
The comments above the changed lines should be updated to reflect the verbiage change.